pal/screen-adapter/web/screen-adapter.ts

```diff
@@ -392,21 +402,49 @@ class ScreenAdapter extends EventTarget {
                 }, { once: true });
             };
             updateDPRChangeListener();
-        }
-        window.addEventListener('orientationchange', (): void => {
-            if (this._orientationChangeTimeoutId !== -1) {
-                clearTimeout(this._orientationChangeTimeoutId);
-            }
-            this._orientationChangeTimeoutId = setTimeout((): void => {
-                if (!this.handleResizeEvent) {
-                    return;
+            
+            const mediaQueryPortrait = window.matchMedia('(orientation: portrait)');
+            const mediaQueryLandscape = window.matchMedia('(orientation: landscape)');
+            const handleOrientationChange = (): void => {
+                let tmpOrientation = adapter._orientation;
+                const orientationType = screen.orientation.type;
+                if (mediaQueryPortrait.matches) {
+                    tmpOrientation = Orientation.PORTRAIT;
```

Comment:
Not PORTRAIT_UPSIDE_DOWN?

Reply:
Currently verified on browsers such as Huawei and Chrome, the orientation cannot be reversed on the web.